parquet-avro/src/main/java/org/apache/parquet/avro/AvroReadSupport.java

```diff
@@ -44,16 +44,15 @@ public class AvroReadSupport<T> extends ReadSupport<T> {
 
   private static final Logger LOG = LoggerFactory.getLogger(AvroReadSupport.class);
 
-  public static String AVRO_REQUESTED_PROJECTION = "parquet.avro.projection";
+  public static final String AVRO_REQUESTED_PROJECTION = "parquet.avro.projection";
```

Comment:
All of these newly `final` fields should be excluded by the japicmp configuration.